Packs/FeedAzureADConnectHealth/Integrations/AzureADConnectHealthFeed/AzureADConnectHealthFeed.py

```diff
@@ -0,0 +1,198 @@
+import demistomock as demisto
+from CommonServerPython import *
+
+from typing import Any, Callable, Dict, List, Tuple
+
+import urllib3
+from bs4 import BeautifulSoup
+import re
+
+# disable insecure warnings
+urllib3.disable_warnings()
+
+INTEGRATION_NAME = 'Microsoft Azure Additional Feed'
+
+
+class Client(BaseClient):
+    """
+    Client to use in the Microsoft Azure Feed integration. Overrides BaseClient.
+    """
+
+    def __init__(self, base_url: str, verify: bool = False, proxy: bool = False):
+        """
+        Implements class for Microsoft Azure feeds.
+        :param url: the Azure endpoint URL
+        :verify: boolean, if *false* feed HTTPS server certificate is verified. Default: *false*
+        :param proxy: boolean, if *false* feed HTTPS server certificate will not use proxies. Default: *false*
+        """
+        super().__init__(base_url, verify=verify, proxy=proxy)
+
+    def build_iterator(self) -> List:
+        """Retrieves all entries from the feed.
+        Returns:
+            A list of objects, containing the indicators.
+        """
+        result = []
+        r = self._http_request('GET', url_suffix='', full_url=self._base_url, resp_type='text')
+
+        soup = BeautifulSoup(r, 'html.parser')
+
+        def subs(text):
```

Comment:
If you can add a docstring it would be very helpful

Reply:
Done